src/purchase-client.ts

```diff
@@ -177,15 +178,47 @@ export class DevPurchaseClient implements PurchaseClient {
       const products = await this.getProducts();
       const item = products.find((p: Product) => p.sku === sku);
 
+      const testDate = new Date();
+
+      const jwtHeader = {
+        "alg": "HS256",
+        "typ": "JWT"
+      };
+
+      const jwtPayload = {
+        "topic": "bits_transaction_receipt",
+        "exp": Math.round(testDate.getMilliseconds() + 10000 / 1000),
```

Comment:
Why the `10000 / 1000`? Did you want `(testDate.getMilliseconds() + 10000) / 1000`?

Also, I believe you want `testDate.valueOf()` here. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMilliseconds

Reply:
Ah.  yes on both.